docs/api_documentation.md

```diff
@@ -0,0 +1,45 @@
+# API Docs
+## Generating Docs
+Docs are autogenerated based off of the requests specs, no extra code is needed.
+
+The specs are run with the OPENAPI env var set to trigger the doc generation, it is set on pushes to main and when changes are made to `spec/requests/api`.
+An autolabeler action takes care of labeling for any PR that changes the `spec/requests/api` files.
+
+*The label has to be created before hand for the labeler to work*
+
+### Parallelization
+All request specs are ran parallelized in different nodes with N threads each. Each thread produces a piece of the updated doc file, which we then merge and upload. After each node uploads its piece of the update doc we download all of them and merge them together into the final file.
+
+### Removing CI action
```

Comment:
the diffs below might change after changes are done

Reply:
I will remove the code and specify the job/steps that need removing, so we don't have to face those changes